### PR TITLE
Add a guide for deploying the Grafana sandbox.

### PR DESCRIPTION
**Pull request by bcipriano**

### CODE REVIEW

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+*   Grafana to provide an interface for querying and displaying the data send by the others.
+
+## Before you begin
+
+*  Follow our quick start guide, which will walk you through standing up a basic OpenCue
+   deployment using Docker compose. This guide builds on the quick start and they share the same
+   prerequisites.
+*  Install the [Loki Docker driver](https://grafana.com/docs/loki/latest/clients/docker-driver/).
+
+## Deploying the OpenCue with monitoring sandbox environment
```
Comment (sharifsalah) [inline content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md:26]: 'the OpenCue' reads a little awkwardly to me. I think the description version is clearer, e.g.:

Deploying OpenCue with external monitoring services

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+The sandbox deployment described here utilizes:
+
+*   Prometheus for collecting metrics from the OpenCue database and scheduler.
+*   Loki for collecting logs from all OpenCue components.
+*   Grafana to provide an interface for querying and displaying the data send by the others.
+
+## Before you begin
+
+*  Follow our quick start guide, which will walk you through standing up a basic OpenCue
```
Comment (sharifsalah) [inline content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md:21]: Maybe add a link to /docs/quick-starts/ here for people who aren't familiar with the quick starts.

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+To view the data exported by the monitoring stack, log into the Grafana endpoint provided by the
+sandbox environment.
+
+1.   Open your browser and visit <http://localhost:3000/>.
+2.   Log in with the default admin account:
+       *   User: admin
+       *   Password: admin
+3.   Change the `admin` password if prompted.
+
+Dashboards > Manage will show the sample dashboards included with the sandbox:
```
Comment (sharifsalah) [inline content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md:73]: I missed this the first time I read through, so maybe add it to the list:

4.   To view the sample dashboards included with the sandbox, click **Dashboards** > **Manage**.

**Review by sharifsalah — APPROVED** [review]
LGTM overall! I left a few minor comments.

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md (hunk 1 above)
Comment (bcipriano) [reply content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md]: Agree, struggled to find the best wording here. I like yours, done.

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md (hunk 2 above)
Comment (bcipriano) [reply content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md]: Meant to do that, done.

File: content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md (hunk 3 above)
Comment (bcipriano) [reply content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md]: Done.

**Pull request merged**